M2/Macaulay2/packages/Complexes/ChainComplexDoc.m2

```diff
@@ -4363,6 +4372,66 @@ doc ///
         freeResolution
 ///
 
+doc ///
+    Key
+        (Ext,Module,Module)
+        (Ext,Ideal,Ideal)
+        (Ext,Ideal,Module)
+        (Ext,Ideal,Ring)
+        (Ext,Module,Ideal)
+        (Ext,Module,Ring)
+    Headline
+        total Ext module
+    Usage
+        Ext(M, N)
+    Inputs
+        M:Module
+            or ofClass{Ideal,Ring}, that is homogeneous
+        N:Module
+            or ofClass{Ideal,Ring} over the same ring as $M$, that is also homogeneous
+    Outputs
+        :Module
+            the $\operatorname{Ext}$ module of $M$ and $N$, as a
```

Comment:
In principle, I'm in favor of using latex in the documentation, but this is also displayed in emacs/terminal and it doesn't display well there. We could define macros for `\Ext` (and `\Hom`, `\Tor`, etc.) in the header in `m2/html.m2` though.

We can also actively decide that we don't care about how the documentation looks in emacs/terminal, but I don't think that's the consensus, right?

Reply:
For the moment, we will leave this in.  We might want a more systematic addition of macros to katex, or we might want to have some of the tex functions become no-ops in terminal/emacs output.  I'm not sure which is best.

Reply:
It's possible to render TeX in Emacs (see https://github.com/Macaulay2/M2/wiki/Previewing-TeX-in-Macaulay2-Emacs-buffers), so I'd rather not make them no-ops.

Reply:
Can the output of `help` be rendered as TeX automatically?